ensembl/htdocs/info/genome/variation/species/populations.html

```diff
@@ -279,6 +280,45 @@ <h3>Populations from <a href="https://www.uk10k.org/" target="_blank" style="tex
 </table>
 
 
+  <div style="margin-bottom:35px"></div>
+
+      <h3>Populations from  the gambian_genome_variation_project</h3>
```

Comment:
This would look better as 'Populations from the Gambian Genome Variation Project'